dll/cpl/input/lang/cs-CZ.rc

```diff
@@ -10,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Nastavení"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Default input &language", -1, 5, 5, 246, 55
-    LTEXT "Select one of the installed input languages to use when your computer starts.", -1, 15, 15, 230, 23
+    GROUPBOX "Výchozí vstupní &jazyk", -1, 5, 5, 246, 55
+    LTEXT "Vyberte jeden z nainstalovaných jazyků, který se bude používat po spuštění Vašeho počítače.", -1, 15, 15, 230, 23
```

Comment:
The (longer) string still fits OK in the window?

Reply:
@HBelusca I would say, yes.
![Screenshot_live_test_2024-07-06_22:50:53](https://github.com/reactos/reactos/assets/13459614/f18510db-c152-4cdb-bf2a-050548ad10a2)

Reply:
Resolved.